lib/promise-webdriver.js

```diff
@@ -169,6 +169,14 @@ module.exports = function(WebDriver, Element, chainable) {
               throw new Error("Invalid method " + fname);
             }
 
+            // Report error if method in requested scope is not available.
+            if (!isBrowserMethod && (scopeHint === '<')) {
+              throw new Error("No method in browser scope " + fname);
+            }
+            if (!isElementMethod && (scopeHint === '>')) {
+              throw new Error("No method in element scope " + fname);
+            }
+
```

Comment:
I have not completely investigated the logic there, but just below there is a block for  "resolving the conflict (between isBrowserMethod and isElementMethod)", what makes me believe that your addition before that block could cause "false error cases" or lose them...

Reply:
Ahh, you want to catch the "mismatch" before it's "resolved"?! Sorry, I was wrong.